packages/clay-drop-down/src/index.tsx

```diff
@@ -0,0 +1,149 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+import {Align} from 'metal-position';
+import {createPortal} from 'react-dom';
+import {useDropdownCloseInteractions} from './hooks';
+
+import Action from './Action';
+import Caption from './Caption';
+import Divider from './Divider';
+import Group from './Group';
+import Help from './Help';
+import Item from './Item';
+import Search from './Search';
+
+const Portal = ({children}: {children: React.ReactNode}) => {
+	const portalRef = React.useRef(document.createElement('div'));
+	const elToMountTo = document && document.body;
+
+	React.useEffect(() => {
+		elToMountTo.appendChild(portalRef.current);
+		return () => {
+			elToMountTo.removeChild(portalRef.current);
+		};
+	}, [elToMountTo]);
+
+	return createPortal(children, portalRef.current);
+};
+
+interface DropDownMenuProps extends React.HTMLAttributes<HTMLUListElement> {
+	alignElement: HTMLElement | null;
+	hasLeftSymbols?: boolean;
+	hasRightSymbols?: boolean;
+	ref?: React.Ref<HTMLUListElement>;
+}
+
+const DropDownMenu: React.ComponentType<DropDownMenuProps> = React.forwardRef(
+	(
+		{
+			alignElement,
+			children,
+			hasLeftSymbols,
+			hasRightSymbols,
+			...otherProps
+		},
+		ref?: React.Ref<HTMLUListElement>
+	) => {
+		return (
+			<ul
+				{...otherProps}
+				className={classNames('dropdown-menu list-unstyled show', {
```

Comment:
As I had said in the other comments, we should have inside the `ul` only `li` tags.

For example:
```html
<div aria-labelledby="theDropdownToggleId" class="dropdown-menu">
	<div class="alert alert-fluid alert-info" role="alert">
		You can customize this menu or see all you have by pressing "more".
	</div>
	<ul class="list-unstyled">
		<li><a class="dropdown-item" href="#1">D Structure</a></li>...
	</ul>
	<div class="dropdown-caption">Showing 7 of 203 Structures</div>
	<div class="dropdown-section">
		<button class="btn btn-block btn-secondary">More</button>
	</div>
</div>
```